dist/kotlin/src/test/kotlin/ValidatorTests.kt

```diff
@@ -0,0 +1,22 @@
+import com.loginradius.EmailValidator
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+object ValidatorTests : Spek( {
+
+    describe("The validator") {
+        mapOf(
+            "valid@gmail.com" to true,
+            "valid@yahoo.com" to true,
+            "qwerty" to false,
+            "asdf@asdf@asdf" to false,
```

Comment:
@desiderantes 
Please fix  "," from the last line it is causing an error

Reply:
Fixed